.github/workflows/linux.yml

```diff
@@ -42,6 +45,12 @@ jobs:
           echo 'deb https://ppa.launchpadcontent.net/git-core/ppa/ubuntu bionic main' > /etc/apt/sources.list.d/git.list
           apt update
 
+      - if: ${{ env.ACT }}
```

Comment:
Another leftover step?

Reply:
I used this for debugging with https://github.com/nektos/act, will remove. It didn't really work that well anyway.